### ISSUE TITLE
Information regarding file policies

### ISSUE THREAD

**Issue by SecuRudy** [MicrosoftDocs/CloudAppSecurityDocs#588]
Dear,

I've been playing around with file policies myself, and tried to follow the following advice in the article.
"When using the file policy filters, Contains will search only for full words – separated by commas, dots, or spaces to search.

Spaces between words function like OR. For example, if you search for malware virus it will find all files with either malware or virus in the name, so it will find both malware-virus.exe and virus.exe."

I tried doing this myself, but none of the options work.
Is there a bug that prevents this in Microsoft Defender for Cloud Apps?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f258bad7-5c1a-ad1d-ee3b-a0298a49d855
* Version Independent ID: c254079f-f5d9-bac4-2966-bcbb8031f1c9
* Content: [Understanding file data and filters available in Defender for Cloud Apps](https://docs.microsoft.com/en-us/defender-cloud-apps/file-filters)
* Content Source: [CloudAppSecurityDocs/file-filters.md](https://github.com/Microsoft/CloudAppSecurityDocs/blob/main/CloudAppSecurityDocs/file-filters.md)
* Service: **defender-for-cloud-apps**
* GitHub Login: @dcurwin
* Microsoft Alias: **dacurwin**

**Comment by dcurwin** [comment]
This isn't a documentation issue and requires further investigation. The best way to get help is to open a support ticket.

**Comment by SecuRudy** [comment]
Hi dcurwin, 
I indeed opened a support ticket, but played around with it some more.
I found how to make it working, maybe it helps for people with the same issue.

The "OR" functionality seems to work with quotation marks, but without spaces. 
e.g. ("hello""world") will find files with only the single word "hello", "world" and files with either of the words in it. 
("hello" "world") would not find the files with the single words. 

Hope it helps :)
